scripts/dev/run-mailpit.sh

```diff
@@ -0,0 +1,109 @@
+#!/bin/bash
```

Comment:
Its also available as a docker container, in the interest of security and isolation it seems more suitable than installing a binary that run as the current user. And it works on all platforms.
https://mailpit.axllent.org/docs/install/docker/